core/templates/pages/preferences-page/form-fields/subject-interests.component.ts

```diff
@@ -148,4 +148,26 @@ export class SubjectInterestsComponent implements ControlValueAccessor {
       interest.toLowerCase().includes(filterValue)
     );
   }
+
+  onBlur(): void {
+    const value = this.subjectInterestInput.nativeElement.value.trim();
+
+    if (value && this.validInput(value)) {
+      this.subjectInterests.push(value);
+      this.subjectInterestInput.nativeElement.value = '';
```

Comment:
Just for understanding -- what does nativeElement.value represent (on the page)?

Some comments here might be helpful.